tests/containers/workbenches/workbench_image_test.py

```diff
@@ -31,21 +32,48 @@ class TestWorkbenchImage:
         # disable ipv6 https://danwalsh.livejournal.com/47118.html
         {"net.ipv6.conf.all.disable_ipv6": "1"}
     ])
-    def test_image_entrypoint_starts(self, image: str, sysctls) -> None:
+    def test_image_entrypoint_starts(self, subtests: pytest_subtests.SubTests, image: str, sysctls) -> None:
         skip_if_not_workbench_image(image)
 
+        # Here is a list of blocked keywords we don't want to see in the log messages during the container/workbench
```

Comment:
maybe put all this into a helper function (logs and subtests object can be a parameter to it) and also call it from the ipv6 only test below?